.env.dist

```diff
@@ -3,5 +3,6 @@ APP_DEBUG=1
 
 APP_KEY=base64:c9yprqAy56EwJ1pvPeyRkP6xV3MTtsa498Hsffd3/rw=
 APP_URL=http://localhost
+ASSET_URL=/
```

Comment:
there are 2 `.env`:

```
.env.dist
.env.prod
```

this may need update `.env.prod` as well for production environment.